lib/va_profile/v2/contact_information/service.rb

```diff
@@ -35,11 +35,11 @@ class Service < VAProfile::Service
         def get_person
           with_monitoring do
             vet360_id_present!
-            raw_response = perform(:get, "#{MPI::Constants::VA_ROOT_OID}/#{ERB::Util.url_encode(icn_with_aaid)}")
+            raw_response = perform(:get, "#{MPI::Constants::VA_ROOT_OID}/#{ERB::Util.url_encode(uuid_with_aaid)}")
             PersonResponse.from(raw_response)
           end
         rescue Common::Client::Errors::ClientError => e
-          if e.status == 400
+          if e.status == 404
```

Comment:
Do we need to still cover the 400 status code here?

Reply:
The 400 is covered below.